src/app/Common/ConsoleBackgroud.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {BackgroundImage} from '@patternfly/react-core';
+import back1 from '!!url-loader!@app/assets/images/infinispanbg_576.png';
```

Comment:
Couldn't you import the images as `lg`, `sm` etc., to avoid the extra backX names?

Reply:
yes